typing/parmatch.ml

```diff
@@ -1619,7 +1616,7 @@ let extract_columns pss qs = match pss with
    check or-patterns argument usefulness (terminal case)
 *)
 
-let rec every_satisfiables pss qs = match qs.active with
+let rec every_satisfiables pss qs = let open Usefulness in match qs.active with
```

Comment:
:/

Reply:
I generally try to avoid changing the indentation level from a patch, but here we can change it if we review with whitespace ignored anyway...